resources/views/layouts/_nav.blade.php

```diff
@@ -10,15 +10,15 @@
 
                         <div class="flex lg:hidden">
                             <button @click="showSearch($event)">
-                                <x-heroicon-o-search class="w-6 h-6 mr-4" />
+                                <x-heroicon-o-magnifying-glass class="w-6 h-6 mr-4" />
                             </button>
 
                             <button @click="nav = !nav">
-                                <x-heroicon-o-menu-alt-1 x-show="!nav" class="w-6 h-6" />
+                                <x-heroicon-m-bars-3-center-left x-show="!nav" class="w-6 h-6" />
```

Comment:
Are you sure we need the mini icons? Isn't there an outline version?

Reply:
Didn't pay attention to it 😅 Well, I changed it to an outline version now